toc/rfc/rfc-draft-bosh-libvirt-cpi.md

```diff
@@ -0,0 +1,41 @@
+# Meta
+[meta]: #meta
+- Name: Golang-based libvirt CPI for BOSH with Multi-Runtime Support
+- Start Date: 2025-06-30
+- Author(s): ZPascal
+- Status: Draft
+- RFC Pull Request: [community#1227](https://github.com/cloudfoundry/community/pull/1227)
+
+## Summary
+
+This proposal introduces a new **Cloud Provider Interface (CPI)** for **BOSH**, implemented in **Go** and based on **libvirt**. It is intended as a modern, maintainable, and extensible alternative to the current **VirtualBox CPI**, supporting multiple virtualization backends, including **VirtualBox**, **QEMU/KVM**, and **container runtimes (e.g., containerd)**. This libvirt-based CPI will serve as a **centralized and runtime-agnostic integration layer** for local or development BOSH deployments, with the initial milestone being to **replace the existing VBox CPI**.
+
+## Problem
+
+The existing VirtualBox CPI:
+- Is minimally maintained and tied to a single backend (VBox).
+- Has limited extensibility and runtime support.
+- Is written in Ruby, with a legacy codebase that is harder to modernize.
```

Comment:
This one is written in golang: https://github.com/cloudfoundry/bosh-virtualbox-cpi-release